src/content/main.js

```diff
@@ -239,19 +239,51 @@ async function addLevelIndicators() {
     if (isUserPage()) {
         var userId = document.querySelector(".page-header h1").innerText.trim()
         var userStaticsTable = document.querySelector("#statics tbody")
-
+        const isShowUserTempTier = JSON.parse(await getPrefs('show_userpage_temp_tier', 'false'))
         const userData = await (await fetch("https://api.solved.ac/user_information.php?id=" + userId)).json()
-        if (!userData) return
 
+        if (!userData &&!isShowUserTempTier) return
+        
         var newRow = document.createElement("tr")
         var newRowHeader = document.createElement("th")
-        newRowHeader.innerText = "solved.ac"
         var newRowDescription = document.createElement("td")
-        newRowDescription.innerHTML = "<a href=\"https://solved.ac/" + userData.user_id + "\">"
-                                        + "<span class=\"text-" + levelCssClass(userData.level) + "\">"
-                                            + levelLabel(userData.level) + "<b>" + userData.user_id + "</b>"
-                                        + "</span>"
-                                        + "</a>"
+
+        if(!userData && isShowUserTempTier)
+        {
+            var acceptPanel = document.querySelector(".panel.panel-default")
+            var acceptProblems = acceptPanel.querySelectorAll(".problem_number>a")
```

Comment:
Since BOJ and solved.ac calculates accepted problems differently, the actual experience points calculated from this method will differ from official calculation. See https://solved.ac/faq/

Reply:
Could you explain more details?
I'm not sure what I should change on this part.

Reply:
If problem is marked with tag 'Preparing Judge', it should not be calculated as accepted.

Reply:
Does api server  response level 0 if the problem that was tagged 'preparing judge'?